zc_common/jwt_auth/authentication.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, **kwargs):
             setattr(self, kwarg, kwargs[kwarg])
 
     def is_authenticated(self):
-        return 'user' in self.roles or 'service' in self.roles
+        return True
```

Comment:
Very well then. Let's have it always return `True`, but add a comment saying that checking for users with the anonymous role happens in permissions. That way no one else gets confused like I did.